deployment/hasura/metadata/databases/AerieCommanding/tables/public_user_sequence.yaml

```diff
@@ -0,0 +1,3 @@
+table:
+  name: user_sequence
+  schema: public
```

Comment:
We should add in the cross-db association to the command dictionary (not necessary in this PR though)

Reply:
Don't forget to create a ticket for this

Reply:
Agree. This would be just a regular intra-db relationship right since the command_dictionary table exists in the same database as the user_sequence? Happy to add in for this PR since it's simple.

Reply:
Done